metrics/size.table/table.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2022 Block, Inc.
+
+package sizetable
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/cashapp/blip"
+	"github.com/cashapp/blip/sqlutil"
+)
+
+const (
+	DOMAIN = "size.table"
+
+	OPT_ALL = "all"
+
+	TABLE_SIZES_QUERY = `
+    SELECT table_schema AS db, table_name as tbl,
+           data_length + index_length AS tbl_size_bytes
+      FROM information_schema.TABLES
+     WHERE table_schema NOT IN ('performance_schema', 'information_schema', 'mysql');`
```

Comment:
Make this an option (`OPT_SCHEMA_FILTER` perhaps?) with default value equal to this + `sys`.